Description: Also packages Alembic migration scripts
 Thanks to Waldi for the bug report.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-08-28

--- neutron-fwaas-2015.1.0.orig/MANIFEST.in
+++ neutron-fwaas-2015.1.0/MANIFEST.in
@@ -7,6 +7,7 @@ include neutron/db/migration/alembic.ini
 include neutron/db/migration/alembic_migrations/script.py.mako
 include neutron/db/migration/alembic_migrations/versions/README
 recursive-include neutron/locale *
+recursive-include neutron_fwaas/db *
 
 exclude .gitignore
 exclude .gitreview
